addons/l10n_in/views/res_partner_views.xml

```diff
@@ -10,8 +10,9 @@
                 <attribute name="attrs">{'required':[('l10n_in_gst_treatment', 'in', ['regular', 'composition', 'special_economic_zone', 'deemed_export'])], 'readonly': [('parent_id', '!=', False)]}</attribute>
             </xpath>
             <xpath expr="//field[@name='vat']" position="before">
-                <field name="l10n_in_gst_treatment" attrs="{'readonly': [('parent_id', '!=', False)]}"/>
+                <field name="fiscal_country_code" invisible="1"/>
+                <field name="l10n_in_gst_treatment" attrs="{'readonly': [('parent_id', '!=', False)], 'invisible': [('fiscal_country_code', '!=', 'IN')]}"/>
```

Comment:
```suggestion
                <field name="l10n_in_gst_treatment" attrs="{'readonly': [('parent_id', '!=', False)], 'invisible': [('fiscal_country_code', '!=', 'IN')]}"/>
```
<field name="l10n_in_gst_treatment" attrs="{'readonly': [('parent_id', '!=', False)], 'invisible': ['!', ('fiscal_country_codes', 'like', 'IN')]}"/>